### DISCUSSION TITLE
Avoid replying a reply

### DISCUSSION THREAD

**Discussion by elbeto87** [tweepy/tweepy#1764] [Questions] [answered]
Hi all!

I cannot find in the documentation if there is a way to avoid replying or twitting in a thread.

For example:
Case 1- If you mention my bot, I will reply you, and that's it, I dont want to reply a new reply
Case 2- If my twitbot tweets and someome replies, I dont want to reply a reply neither

Thanks!

**Comment by Harmon758** [comment, marked as answer]
With Twitter API v1.1, you can determine whether a `Status`/[Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) is a reply by checking for the existence of the `in_reply_to_status_id`, `in_reply_to_status_id_str`, `in_reply_to_user_id`, `in_reply_to_user_id_str`, and/or `in_reply_to_screen_name` attributes.

With Twitter API v2, you can use the `referenced_tweets` [Tweet field](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/tweet) or request the `in_reply_to_user_id` Tweet field and check for its existence as an attribute.

**Reply by elbeto87** [reply]
Fantastic thanks!
